js/logic.js

```diff
@@ -43,4 +43,32 @@ button.addEventListener('click', () => {
   render();
 });
 
+const dt = new Date();
+document.getElementById('date-time').innerHTML = dt;
+
+const book = document.getElementById('book');
```

Comment:
- [ ] [OPTIONAL] Your app doesn't add the book to the list, it produces an error in the console. kindly consider fixing it before merging the changes so your app works as expected

![Screenshot_20220803_173708](https://user-images.githubusercontent.com/25227022/182662567-5ad1771d-a2bb-4004-89d4-7bc382f7d86f.png)